code/modules/atmospherics/machinery/airalarm/area.dm

```diff
@@ -0,0 +1,8 @@
+// all air alarms, vents, sensors, and scrubbers in area are connected via m̶a̶g̶i̶c̶  bluespess
+// TODO: Refactor to use NTNet once https://github.com/tgstation/tgstation/pull/53368 is done
+/area
+	var/datum/airalarm_control/air_control
```

Comment:
Shouldn't this go in code/game/area/areas.dm? Definitions are usually consolidated in one place.

Reply:
Also autodoc.

Reply:
Theoretically yeah, but I think this can stay for the time being as it's slated to be removed. Will port this to NTNet as soon as that's done